test-runner/utils/test.py

```diff
@@ -57,7 +57,7 @@ class Test(BaseModel):
     notebook: Optional[bool] = False
     serving: Optional[bool] = False
     cap_add: Optional[str] = "AUDIT_READ"
-    device: Optional[str] = []
+    device: Optional[List[str]] = []
```

Comment:
Can you update the test-runner README to reflect the changes to the schema?

Reply:
updated